src/collection/collection.helpers.tsx

```diff
@@ -32,47 +37,72 @@ type ValidationRule<T> = {
 
 const VALIDATION_RULES_FOR_SAVE: ValidationRule<Partial<Avo.Collection.Collection>>[] = [
 	{
-		error: 'De collectie beschrijving is te lang',
+		error: collection =>
+			collection.type_id === ContentTypeNumber.collection
+				? i18n.t('De collectie beschrijving is te lang')
+				: i18n.t('De bundel beschrijving is te lang'),
```

Comment:
Het* bundel

Ze moet niet denken dat we geen Nederlands kunnen. 😉

Reply:
een beetje vroeger gaan slapen benji

![image](https://user-images.githubusercontent.com/1710840/73927934-921e1b00-48d2-11ea-94d9-38f38d2245c6.png)

![image](https://user-images.githubusercontent.com/1710840/73927935-934f4800-48d2-11ea-8086-1d71ee43434b.png)

de "De" slaat trouwens op het woord "beschrijving"